cumulusci/tasks/package_2gp.py

```diff
@@ -445,6 +433,7 @@ def _create_version_request(
             "SkipValidation": skip_validation,
             "Tag": f"hash:{package_hash}",
             "VersionInfo": version_info,
+            "CalculateCodeCoverage": not skip_validation,
```

Comment:
I suspect this also needs to be off when creating an org-dependent unlocked package.

Reply:
Based on https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_unlocked_pkg_code_coverage.htm it sounds like it's not supported for org-dependent unlocked packages, but also not invalid to specify it. So I guess I'll resolve this.